web/src/components/tabs/Tabs/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { ReactElement, useState } from 'react'
+import { useTheme } from 'react-jss'
+
+import { Tab } from '../Tab'
+
+import useStyles from './styles'
+
+export type TabsProps = {
+  size?: 'large' | 'regular' | 'small'
+  align?: 'left' | 'right' | 'justify'
+  margin?: number
+  width?: number
+  children: Array<ReactElement>
+}
+
+export const Tabs: React.FC<TabsProps> = props => {
+  const theme = useTheme()
+
+  const classes = useStyles({
+    ...props,
+    theme,
+  })
+
+  const { children, size, align } = props
+
+  const [active, setActiveTab] = useState(0)
+
+  const onChangeTab = (tab: number) => {
+    if (tab < 0 || tab > children.length - 1) return
+    setActiveTab(tab)
+  }
+
+  return (
+    <div>
```

Comment:
do we really need that extra div wrapped around? please remove!

Reply:
Removed, thanks. Added this to push blocks in stories to a new line :D